subsys/bluetooth/services/alexa_gadgets/gadgets_profile.c

```diff
@@ -1173,9 +1166,31 @@ int gadgets_profile_adv_start(bool bondable_type)
 	}
 }
 
-int gadgets_profile_custom_event_json_send(uint8_t *name, uint8_t *json)
+void bt_gadgets_profile_adv_stop(void)
+{
+	if (atomic_clear(&profile_active)) {
+		(void) bt_le_adv_stop();
+	}
+}
+
+void bt_gadgets_profile_mtu_exchange_performed(struct bt_conn *conn)
```

Comment:
This could be shortened to `bt_gadgets_profile_mtu_exchanged`

Reply:
Shortened as suggested